tardis/io/config_reader.py

```diff
@@ -486,7 +487,9 @@ def parse_spectrum_list2dict(spectrum_list):
     """
     Parse the spectrum list [start, stop, num] to a list
     """
-
+    if isinstance(spectrum_list, dict):
```

Comment:
I was expecting a comment on this line.
I even thought about adding a "Dear Wolfgang, this is as temporary as it can get, please act like you didn't notice the missing ducks" comment. :stuck_out_tongue: 

I'll do the change...